client/components/thank-you/index.tsx

```diff
@@ -160,22 +156,32 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 
 					{ showSupportSection && (
 						<ThankYouSectionContainer>
-							<ThankYouHeaderTitle className="thank-you__body-header wp-brand-font">
+							<ThankYouSectionTitle className="thank-you__body-header wp-brand-font">
 								{ translate( 'How can we help?' ) }
-							</ThankYouHeaderTitle>
-							<p>
+							</ThankYouSectionTitle>
+							<p className="thank-you__help-title">
 								{ translate(
 									'Our Happiness Engineers are here if you need help, or if you have any questions.'
 								) }
 							</p>
-							<VerticalNav>
-								<VerticalNavItem path={ CALYPSO_CONTACT }>
-									{ translate( 'Ask a question' ) }
-								</VerticalNavItem>
-								<VerticalNavItem path={ SUPPORT_ROOT }>
-									{ translate( 'Support documentation' ) }
-								</VerticalNavItem>
-							</VerticalNav>
+							<a
+								className="thank-you__help-link"
+								href={ CALYPSO_CONTACT }
+								target="_blank"
+								rel="noreferrer noopener"
+							>
+								{ translate( 'Ask a question' ) }
+								<Gridicon className="thank-you__icon-external" icon="external" />
+							</a>
+							<a
+								className="thank-you__help-link"
+								href={ SUPPORT_ROOT }
+								target="_blank"
+								rel="noreferrer noopener"
+							>
+								{ translate( 'Support documentation' ) }
```

Comment:
Nit: the designs have the text here as `'View support documentation'`, which is different than what we had before.